src/theme/palette.js

```diff
@@ -0,0 +1,37 @@
+export const palette = {
```

Comment:
This pattern is pretty common so we define all the colors we want in one place and then pick them from this object as we need them. Again we could move the entire theme into css if we wanted.